doc-content/shared-kie-docs/src/main/asciidoc/Workbench/SecurityManagement/configuring-EAP-security-management-provider-CLI-mode-proc.adoc

```diff
@@ -0,0 +1,44 @@
+[id='configuring-EAP-security-management-provider-CLI-mode-proc_{context}']
+
+= Configuring the {EAP} security management provider based on property files and CLI mode
+
+To use the {EAP} security management provider based on property files and CLI mode, complete the following steps:
+
+.Prerequisites
+
+* {EAP} is installed.
+
+.Procedure
+
+. To use an existing users or roles property files from the {EAP} instance, include the following system properties in the `EAP_HOME/standalone/configuration/application-users.properties` and `EAP_HOME/standalone/configuration/application-roles.properties` files:
```

Comment:
property file

Also, I would like to see an example. Show me how these properties will look in the files, then you can follow with the table. This isn't a show stopper but would be nice to do.